### ISSUE TITLE
No sound on linux self compiled build.

### ISSUE THREAD

**Issue by KerDelos** [nesbox/TIC-80#796]
On Fedora 28, with commit https://github.com/nesbox/TIC-80/commit/e016565097d39ce00b04ad2879289225e9f8b562, there is no sound at all on the TIC80 and TIC80pro bin that I compiled.

Sound works as expected with TIC80-sokol and TIC80pro-sokol though.

**Comment by jdmarble** [comment]
Same symptoms here on Ubuntu 18.10.

**Comment by Neoxkc** [comment]
I am having the same issue on Linux as well.
However, unlike mentioned above, I have no sound in any of the bin files.

I am currently running Ubuntu 18.04.

**Comment by farvardin** [comment]
Hello, sound is working fine in TIC-80 when building from sources. 

However, the instructions for compilation are missing some essential packages to build correctly. libasound2-dev is missing, but also some others. I also had the sound problem, so I've added libsdl-sound1.2-dev libsdl-gfx1.2-dev libsdl2-mixer-dev libsdl2-gfx-dev libsdl2-image-dev libsdl-mixer1.2-dev libjack-jackd2-dev (I can't tell which ones are not necessary).

When running cmake, take care of this part:
 ALSA                   (Wanted: ON): ON

Alsa should be on. First time for my previous build (without sound), it was OFF. It turned to ON after I've installed the libsdl and libjack devel packages.

I hope it will help you to get sound!
